lib/robots/dor_repo/etd_submit/other_metadata.rb

```diff
@@ -11,51 +11,55 @@ def initialize(opts = {})
           super('etdSubmitWF', 'other-metadata', opts)
         end
 
-        # create metadata datastreams for the work item
+        # create metadata for the work item
         def perform(druid)
-          etd = Etd.find(druid)
+          object_client = Dor::Services::Client.object(druid)
+          etd = object_client.find
```

Comment:
It looks to me like this value gets passed to the *MetadataGenerators, which depend on an ActiveFedora Etd.

Reply:
same question as above.